Assets/Scripts/Room/Minimap/Minimap.cs

```diff
@@ -0,0 +1,893 @@
+using System;
```

Comment:
Minimap is obsolete and deleted. This is the old system. Why re-add this file and make files depend on it?